best-practice-authorization.md

```diff
@@ -421,7 +421,7 @@ in the HTTP GET request that initiates the Websocket handshake defined in
 The Resource Server SHALL validate such tokens in the same manner as it would for a normal
 protected HTTP resources using the HTTP Authorization Request Header.
 
-Further to this, due to limitations in the native JavaScript Websocket API, clients MAY
+Further to this, due to limitations in the native JavaScript WebSocket API, clients MAY
 pass the OAuth2 access token in the query parameters of the request URL during the handshake,
```

Comment:
I'm really sorry for noting these trivial comments one at a time, but I now realise we use "OAuth 2.0" everywhere else in this doc.

Apart from that, I think this PR is now all good from my point of view!! 🤞